### PR TITLE
[2 of 2] Add integration tests for networks

### PR DESCRIPTION
**Pull request by jfurrow**
~~**Relies on the changes in #1996** to correctly compare values in the UI to values from the API response.~~

This PR adds integration tests for the Networks section.

**Checklist**
- [ ] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [x] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

**Comment by jfurrow** [comment]
retest this please

**Comment by MatApple** [comment]
@jfurrow - seems some of the networking tests are failing

File: tests/_fixtures/mesos/overlay-master.json [hunk 1]
```diff
@@ -7,47 +7,47 @@
           "backend": {
             "vxlan": {
               "vni": 1024,
-              "vtep_ip": "198.15.0.1\/20",
+              "vtep_ip": "198.15.0.1/20",
```
Comment (MatApple) [inline tests/_fixtures/mesos/overlay-master.json]: Is this change driven by what the API returns? :)

File: tests/_support/utils/router.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+const _routes = new Map();
```
Comment (MatApple) [inline tests/_support/utils/router.js]: Isn't this code in another PR somewhere? If not, maybe we should split it out into a separate PR

Comment (jfurrow) [inline tests/_support/utils/router.js]: Just rebased this off master, so this should look better now!

File: tests/_fixtures/mesos/overlay-master.json (hunk 1 above)
Comment (jfurrow) [inline tests/_fixtures/mesos/overlay-master.json]: Hmm... I don't remember why I did this actually... let me investigate, perhaps I can revert it.

Comment (jfurrow) [inline tests/_fixtures/mesos/overlay-master.json]: This works fine without the escaped characters, so I reverted it... I'm still not sure why I did that to begin with. Thanks for catching!

**Comment by jfurrow** [comment]
Ping @MatApple @orlandohohmeier

File: tests/_fixtures/mesos/overlay-master.json (hunk 1 above)
Comment (MatApple) [inline tests/_fixtures/mesos/overlay-master.json]: 👍  Cool - thanks @jfurrow!

File: tests/_support/utils/router.js (hunk 2 above)
Comment (MatApple) [inline tests/_support/utils/router.js]: 👍

**Comment by MatApple** [comment]
LGTM, thanks @jfurrow!

File: tests/pages/networking/Networks-cy.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+          getVisibleTableRows($tableRows).forEach(function(tableRow) {
+            const tableCells = tableRow.querySelectorAll("td");
+            const task = fixture.frameworks[0].tasks[0];
+
+            expect(tableCells[0].textContent).to.equal(task.id);
+            expect(tableCells[1].textContent).to.equal(
+              task.statuses[0].container_status.network_infos[0].ip_addresses[0]
+                .ip_address
+            );
+            // TODO: Figure out how to determine the port mapping value from
```
Comment (nLight) [inline tests/pages/networking/Networks-cy.js]: @jfurrow please create a ticket for that.

**Comment by nLight** [comment]
@MatApple I rebased and force pushed thus your review got dismissed. I decided to proceed anyways, since I the code didn't change 🙏
